src/components/CodeBlock.tsx

```diff
@@ -56,15 +56,20 @@ export const CodeBlock = ({
 
   const child = children[index];
 
+  const variables =
+    children[0].language === 'graphql' && children[1]?.label === 'Variables'
```

Comment:
This isn't the convention used by DEMI. Some code blocks are explicitly labeled "Variables" but the majority are not. LOVATO & Scoobie actually depend on the order of the code blocks e.g.:

<https://github.com/seek-oss/scoobie/blob/064474de22b9731b064120d041a04cdf7686dcce/src/components/CodeBlock/CodeChild.tsx#L106-L124>